src/App.asm

```diff
@@ -1,6 +1,69 @@
 .data
 
+str1: .asciiz "Enter total number of entries: "
+str2: .asciiz "Enter an integer: "
+str3: .asciiz "Total positive "
+str4: .asciiz "\nTotal negative "
 
-.globl main
+largest: .word -1
+count: .word 0
+x: .word 0
+y: .word 0
+pos: .word 0
+neg: .word 0
 
-.text
\ No newline at end of file
+.text
+
+la $a0, str1
+li $v0, 4
+syscall
+
+li $v0,5
+syscall
+move $t0, $v0
+sw $t0,count
+
+li $t6,1
+lw $t1, x
+lw $t2, largest
+lw $t3, y
+
+loop_start:
+blt $t1,$t0, loop_body
```

Comment:
variables (`i` and `count`) used in while loop condition must be loaded from memory before such as:
```asm
...
lw  $t1, i
lw  $t0. count
blt $t1, t0, loop_body
...
```